.pylintrc

```diff
@@ -7,7 +7,9 @@ init-import=yes
 
 [MESSAGES CONTROL]
 disable=
-    C0116,  # missing-function-docstring
+    C0114,  # done by pydocstyle
+    C0115,  # done by pydocstyle
+    C0116,  # done by pydocstyle
```

Comment:
do they conflict?

Reply:
Not sure, but all documentation docstrings are handled by pydocstyle now